code/src/components/GameScreen.js

```diff
@@ -0,0 +1,141 @@
+import React from "react";
+import GamePicture from '../assets/GameBoardImage.jpg';
+import { useSelector, useDispatch } from "react-redux";
+import styled from 'styled-components/macro'
+
+import game, { continueGame } from "../reducers/game";
+
+const devices = {
+  mobile: '(min-width: 375px)',
+  tablet: '(min-width: 768px)',
+  desktop: '(min-width: 1024px)'
+}
```

Comment:
I couldn´t agree more, this is great that you set the media quieries like this, in a const for the devices, so it can be reused in the styled components.